pom.xml

```diff
@@ -286,6 +286,9 @@
   <profiles>
     <profile>
       <id>tests</id>
+      <activation>
```

Comment:
I think if you want to make tests run by default then we ought to do away with the tests profile and just add it to the modules list.